static/js/page-specific-scripts-loaded.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Should we name this file/function more generally? Your plan was that this could be used for more than just script tag loading, right? People could define any kind of async task that needs to happen before the `Answers.init`

Reply:
To use this functionality more generally for async tasks, new promises would be added to the `preAnswersInitTasks` array. This function could still be specific to page specific scripts.

Reply:
this is outdated due to switching to use DOMContentLoaded